### PR TITLE
📖 added edge-scheduler to quickstart

### LINKED ISSUE #165: Add a section for the edge-scheduler to quickstart
Right now each moving part of edge-mc (e.g. mailbox-controller, edge-scheduler) have (or will have) their own step-by-step docs written and maintained in [docs/poc2023q1](https://github.com/kcp-dev/edge-mc/tree/main/docs/poc2023q1).

We want to make the life of the first time visitors easier, by linking/migrating the docs to a more explicit entry point, which turns out to be the [quickstart.md](https://github.com/kcp-dev/edge-mc/blob/main/quickstart.md).

### PR DESCRIPTION
**Pull request by nitinmewar**
Signed-off-by: nitinmewar [nitinmewar28@gmail.com](mailto:nitinmewar28@gmail.com)
## Summary
linked edge schedular docs to Quickstart and added table of content.

## Related issue(s)

Fixes #165

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# Table of Contents
```
Comment (waltforme) [inline quickstart.md:1]: Nice table! I like it.

File: quickstart.md [hunk 2]
```diff
@@ -191,7 +211,7 @@ documentation:
 - [Virtual workspaces](https://github.com/kcp-dev/kcp/tree/main/docs/content/en/main/concepts/virtual-workspaces.md) - details on kcp's mechanism for virtual views of workspace content
 
 ### KCP-Edge
-TBD
+- [Edge scheduler](https://github.com/kcp-dev/edge-mc/blob/main/docs/poc2023q1/edge-scheduler.md) - demo of edge scheduler functionality
```
Comment (waltforme) [inline quickstart.md:214]: Maybe better to put this link in some additional section (say "Try KCP-Edge Components") after "Build and Install KCP-Edge"? These components, including the edge scheduler, are in early stage, so their launching are not automated (yet).

@clubanderson I think we can put a link to https://docs.kcp-edge.io/ at this location instead. What do you think?

Comment (clubanderson) [reply quickstart.md:214]: the quickstart.md is linked to by the kcp-edge.io in the https://docs.kcp-edge.io/docs/overview/.  I need to look into how to add a native file that is not in the docs path.  For now, make the necessary adjustments to the root quickstart.md.

File: quickstart.md (hunk 1 above)
Comment (MikeSpreitzer) [reply quickstart.md:1]: This is standard practice in KEPs, and we should follow it too.
See https://github.com/kubernetes/enhancements/blob/master/hack/update-toc.sh for how KEPs do it.

**Review by MikeSpreitzer — COMMENTED** [review]
I am confused.  I have found two copies of this content under source code control (https://github.com/kcp-dev/edge-mc/blob/main/docs/content/en/docs/Getting%20started/quickstart.md and https://github.com/kcp-dev/edge-mc/blob/main/quickstart.md) and neither of then currently says how to exercise kcp-edge, it only starts with a rehearsal of some basic TMC stuff.  Also, there is already a write-up of how to exercise the scheduler, see https://github.com/kcp-dev/edge-mc/blob/main/docs/poc2023q1/edge-scheduler.md .  And there is a write-up of how to exercise the mailbox controller --- see https://github.com/kcp-dev/edge-mc/blob/main/docs/poc2023q1/mailbox-controller.md .

Right now we do not have enough kcp-edge pieces at all working to even suggest we have anything more than two isolated pieces.  The first thing a user finds should be clear about this.  I do not see a need to recapitulate TMC instructions here.

**Comment by clubanderson** [comment]
@MikeSpreitzer yes, your right.  What is the right way to proceed?  Do we need a quickstart in root?  Or can we use just the one in the docs?  I would prefer the docs version.

**Comment by MikeSpreitzer** [comment]
The README can point to the docs, no need for a separate top-level copy of docs content.

I think that at this current stage of development, the README should say that we are working on a PoC and have two pieces that can run and point to their exercise docs.

**Comment by MikeSpreitzer** [comment]
Our README can reference any of the TMC quickstarts in kcp.
Once @pdettori 's blog post is out, our README can reference that --- it has a better scenario for exercising TMC.

**Comment by clubanderson** [comment]
I removed the root quickstart.md.  All references to the quickstart.md from the readme and docs overview page now link to the quickstart at https://docs.kcp-edge.io/docs/getting-started/quickstart/.  @nitinmewar if you would like to update the docs page, simply open the url (https://docs.kcp-edge.io/docs/getting-started/quickstart/) and then click "edit this page" on the right-side menu.  You make changes and submit a PR just like any other document in our source.  Thank you for your help with this.  @waltforme

**Comment by nitinmewar** [comment]
@waltforme what should I do now? should I remove TOC?

**Comment by waltforme** [comment]
@nitinmewar If you are interested :)

Paolo's draft of the blog post is available here:
https://dettori.medium.com/a79eaada5750

Paolo's post presents an interesting scenario for playing with TMC (Transparent MultiCluster).
Some background on TMC, just in case you haven't touch it:
- TMC is currently part of kcp, docs here: https://docs.kcp.io/kcp/main/en/concepts/locations-and-scheduling/.
- But kcp folks are working towards splitting TMC (e.g. https://github.com/kcp-dev/kcp/pull/2795).

File: quickstart.md (hunk 1 above)
Comment (clubanderson) [reply quickstart.md:1]: for now we can do this by hand as @nitinmewar has assembled here.  As we get more advanced/mature we can turn to automation.

**Review by clubanderson — CHANGES_REQUESTED** [review]
we removed the quickstart.md from the root path.  To make a change to the current quickstart you can navigate to https://docs.kcp-edge.io/docs/getting-started/quickstart/.  From there you can click on "edit this page" in the right hand menu.  You can close this PR, and open a new PR with the table you created against the https://docs.kcp-edge.io/docs/getting-started/quickstart/ page.

**Comment by clubanderson** [comment]
closing for now - we are revisiting quickstart in a gh issue #207 and #208

**Pull request closed without merging**